_parts/part03.md

```diff
@@ -63,10 +64,10 @@ Lastly, we add together each of the values we computed for each of the digit pos
 
 It doesn't get more complicated than that! Let's spell this out in steps:
 1. Start at the right-most digit position
-2. Let sum = 0 and base = 0
-3. Multiple the current digit position by $$2^{base}$$
+2. Let sum = 0
+3. Multiple the current digit position by $$2^{value of digit}$$
```

Comment:
Confused by the changes for this conversion algorithm. How does one interpret "value of digit"